scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -139,6 +139,7 @@ class PublishVisibility(Enum):
 STANDARD_ASSET_TYPES = [AssetType.COMPONENT, AssetType.DATA, AssetType.ENVIRONMENT, AssetType.MODEL]
 TEMPLATE_CHECK = re.compile(r"\{\{.*\}\}")
 VERSION_AUTO = "auto"
+DEFAULT_PYTHON_VERSION = "3.11"

```

Comment:
changes to this would require a package release.

Reply:
We are setting python version 3.11 because, azureml-dataprep is not available for python 3.12. I think, it was set as a default after system upgrade.

Reply:
I have removed changes as per offline discussion.